rootfs/python-build/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# https://github.com/revsys/optimized-python-docker/blob/master/Dockerfile
+# https://github.com/docker-library/python/blob/master/3.12/slim-bullseye/Dockerfile
+
+FROM ubuntu:22.04 as base
+
+ENV PATH /usr/local/bin:$PATH
+
+RUN apt update && \
+	apt install -yqq build-essential && \
+	apt install -yqq make && \
+	apt install -yqq curl && \
+	apt install -yqq tar && \
+	apt install -yqq libexpat-dev && \
+	apt install -yqq zlib1g-dev && \
+	apt install -yqq zlib1g-dev
+
+# apt install -yqq libsqlite3-dev && \
+# apt install -yqq libexpat1-dev && \
+# apt install -yqq libffi-dev
+
+ARG PYTHON_VERSION
+ENV PYTHON_VERSION ${PYTHON_VERSION}
+
+ENV SRCDIR /python
+RUN mkdir -p ${SRCDIR}/build && \
+	curl -sL "https://www.python.org/ftp/python/${PYTHON_VERSION}/Python-${PYTHON_VERSION}.tar.xz" | tar -xJC ${SRCDIR} --strip-components=1 -f -
+
+WORKDIR ${SRCDIR}/build
+
+RUN gnuArch="$(dpkg-architecture --query DEB_BUILD_GNU_TYPE)" \
+    &&  ../configure \
+        --build="$gnuArch" \
+        --enable-shared
+
+#        --enable-loadable-sqlite-extensions \
+#        --with-system-expat \
+#        --with-system-ffi
+
+RUN make -j $(( 1 * $( egrep '^processor[[:space:]]+:' /proc/cpuinfo | wc -l ) )) \
+    && make install
+
+RUN find /usr/local -type f -name "*.so" -exec strip --strip-unneeded {} + && \
+	find /usr/local -type f -name "*.so" | ldconfig && \
+	find /usr/local -depth \
+	\( \
+		\( -type d -a \( -name test -o -name tests -o -name __pycache__ \) \) \
+			-o \
+		\( -type f -a \( -name '*.pyc' -o -name '*.pyo' -o -name '*.exe' \) \) \
+	\) -exec rm -rf '{}' +;
```

Comment:
This is not necessary since we end up with a `scratch` container.

Reply:
Currently we're copying the entire `/usr/local/bin/` and `/usr/local/lib/` directories to `scratch`. So we still need to clean them before the copying.